geolib.js

```diff
@@ -703,9 +703,10 @@
 		 *
 		 *  @return     Array [{lat:#lat, lng:#lng, elev:#elev},....]}
 		 */
-		setBusinessSpecificParameters: function(clientId, privateKey){
+		setBusinessSpecificParameters: function(clientId, privateKey, key){
```

Comment:
I left a similar review on the PR in the other repo already, but naming this variable "key" doesn't tell me how this is different from `privateKey`.